fe/fe-core/src/main/java/org/apache/doris/catalog/TableProperty.java

```diff
@@ -137,6 +151,11 @@ public void modifyTableProperties(Map<String, String> modifyProperties) {
         properties.putAll(modifyProperties);
     }
 
+    public void modifyTableProperties(DataSortInfo dataSortInfo) {
```

Comment:
Change the method name

Reply:
modifyTableProperties keeps the same style with before

Reply:
Not same, the origin 2 `modifyTableProperties()` method pass the `key-value`, which means it can modify any of table properties.
But here you just want to modify the data sort info.

Reply:
modify it to modifyDataSortInfoProperties()